### PR TITLE
Updates new functions in @storybook/addon-actions

### PR DESCRIPTION
**Pull request by jicjjang**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/storybook__addon-actions/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-// Type definitions for @storybook/addon-actions 3.0
+// Type definitions for @storybook/addon-actions 3.4
 // Project: https://github.com/storybooks/storybook
-// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions by: Joscha Feth <https://github.com/joscha>, June <https://github.com/jicjjang>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.6
+// TypeScript Version: 2.9
```
Comment (joscha) [inline types/storybook__addon-actions/index.d.ts]: do we need to bump this? The definitions below would work for older TS as well, no?

File: types/storybook__addon-actions/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 export type HandlerFunction = (...args: any[]) => undefined;
 export type DecoratorFunction = (args: any[]) => any[];
 
-export function decorateAction(decorators: DecoratorFunction[]): (name: string) => HandlerFunction;
+export function decorateAction(decorators: DecoratorFunction[]): (name: string, options?: object) => HandlerFunction;
```
Comment (joscha) [inline types/storybook__addon-actions/index.d.ts]: can we make the options more narrow than `object`? There are only three that are allowed: https://github.com/storybooks/storybook/tree/master/addons/actions#available-options

File: types/storybook__addon-actions/index.d.ts [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+export function configureActions(options: object): undefined;
```
Comment (joscha) [inline types/storybook__addon-actions/index.d.ts]: reuse the options type here.

**Review by joscha — CHANGES_REQUESTED** [review]

File: types/storybook__addon-actions/index.d.ts (hunk 1 above)
Comment (jicjjang) [reply types/storybook__addon-actions/index.d.ts]: oh, I see. I will be fix it. Didn't i think about older TS.

File: types/storybook__addon-actions/index.d.ts (hunk 2 above)
Comment (jicjjang) [reply types/storybook__addon-actions/index.d.ts]: I got it :D

**Review by joscha — APPROVED** [review]
:tada:

**Pull request merged**